twitter.js

```diff
@@ -34,17 +30,21 @@ twitter.stream('statuses/filter', {'locations':'-180,-90,180,90'}, function (str
               country: tweet.place.country
             }
           );
-
-          // console.log(text);
-          var unicode = text.map((emoji) => {
-            return '\\u' + emoji.charCodeAt(0).toString(16) + '\\u' + emoji.charCodeAt(1).toString(16);
+          // surrogate pairs: (output like this)
+          // multiple emojis: [ '\\uD83D\\uDE04', '\\uD83D\\uDC96', '\\uD83D\\uDE3B' ]
+          // only one emoji: [ '\\uD83D\\uDE02' ]
+          // these surrogate pairs should match the surrogate pairs in the emoji.json
+          // (if the specific emoji is there)
+          var surrogate = text.map((emoji) => {
+            return '\\u' + emoji.charCodeAt(0).toString(16).toUpperCase() + '\\u' + emoji.charCodeAt(1).toString(16).toUpperCase();
+          });
+          // printing surrogate pairs
+          surrogate.forEach((surrogate) => {
```

Comment:
when I split the surrogate pair it comes in an array with 
```
['','D83D', 'DE02']
```
and after that I use the slice(1)
@thgaskell